scripts/vim-patch.sh

```diff
@@ -179,6 +179,10 @@ preprocess_patch() {
   # Rename path to matchit plugin.
   LC_ALL=C sed -e 's@\( [ab]/runtime\)/pack/dist/opt/matchit/\(plugin/matchit.vim\)@\1/\2@g' \
     "$file" > "$file".tmp && mv "$file".tmp "$file"
+
+  # Rename path to test_urls.vim
+  LC_ALL=C sed -e 's@\( [ab]\)/runtime/doc/\(test_urls.vim\)@\1/scripts/\2@g' \
```

Comment:
Based on the sed command for matchit plugin. Did not test this.

Reply:
Let's name it `check_urls.vim` since we already have `check-includes.py`. Want to avoid unnecessary inconsistency where possible.